package.json

```diff
@@ -0,0 +1,66 @@
+{
+	"name": "@ckeditor/ckeditor4-vue",
+	"version": "0.0.0",
+	"main": "dist/ckeditor.js",
+	"devDependencies": {
+		"@babel/core": "^7.6.4",
+		"@babel/preset-env": "^7.6.3",
+		"@vue/test-utils": "^1.0.0-beta.29",
+		"babel-loader": "^8.0.6",
+		"chai": "^4.1.2",
+		"eslint": "^5.5.0",
+		"eslint-config-ckeditor5": "^1.0.8",
+		"istanbul-instrumenter-loader": "^3.0.0",
+		"karma": "^4.4.1",
+		"karma-browserstack-launcher": "^1.3.0",
+		"karma-chai": "^0.1.0",
+		"karma-chrome-launcher": "^2.2.0",
+		"karma-coverage": "^1.1.1",
+		"karma-coveralls": "^2.0.0",
+		"karma-firefox-launcher": "^1.2.0",
+		"karma-mocha": "^1.3.0",
+		"karma-mocha-reporter": "^2.2.4",
+		"karma-sinon": "^1.0.5",
+		"karma-sourcemap-loader": "^0.3.7",
+		"karma-webpack": "^3.0.5",
+		"live-server": "^1.2.1",
+		"minimist": "^1.2.0",
+		"mocha": "^5.2.0",
+		"sinon": "^7.5.0",
+		"terser-webpack-plugin": "^1.1.0",
+		"vue": "^2.5.17",
+		"vue-router": "^3.1.3",
+		"vue-template-compiler": "^2.5.17",
+		"webpack": "^4.41.2",
+		"webpack-cli": "^3.3.9"
+	},
+	"peerDependencies": {
+		"vue": "^2.5.17"
+	},
+	"engines": {
+		"node": ">=8.0.0",
+		"npm": ">=5.7.1"
+	},
+	"scripts": {
+		"start": "npm run build && live-server --open=samples/index.html",
+		"build": "webpack --mode production",
+		"develop": "webpack --mode development --watch",
+		"test": "karma start",
+		"preversion": "npm test",
+		"version": "npm run build && git add -f dist/",
+		"postversion": "git rm -r --cached dist/ && git commit -m \"Clean after release [ci skip]\" && git push origin && git push origin --tags"
+	},
+	"repository": {
+		"type": "git",
+		"url": "https://github.com/ckeditor/ckeditor4-vue.git"
+	},
+	"author": "CKSource (http://cksource.com/)",
+	"license": "(GPL-2.0-or-later OR LGPL-2.1-or-later OR MPL-1.1-or-later)",
+	"bugs": {
+		"url": "https://github.com/ckeditor/ckeditor4-vue/issues"
+	},
+	"homepage": "https://github.com/ckeditor/ckeditor4-vue",
+	"dependencies": {
+		"load-script": "^1.0.0"
+	}
```

Comment:
I know that it's not much important, but… can we move this section to others dependencies? It's triggering me.

Reply:
Not sure if I understand :thinking: Where would you like to move this dependency specifically?

Reply:
Not the dependency, but whole section, to group it with other deps:

```json
"devDependencies": {},
"peerDependencies": {},
"dependencies": {}
```